app/app/controllers/cbv/payment_details_controller.rb

```diff
@@ -44,50 +43,62 @@ def account_comment
 
   private
 
+  def has_income_data?
+    @pinwheel_account.supported_jobs.include?("income") && @pinwheel_account.income_errored_at.blank?
+  end
+
+  def has_employment_data?
+    @pinwheel_account.supported_jobs.include?("employment") && @pinwheel_account.employment_errored_at.blank?
+  end
+
+  def has_paystubs_data?
+    @pinwheel_account.supported_jobs.include?("paystubs") && @pinwheel_account.paystubs_errored_at.blank?
+  end
+
   def employer_name
-    @employment["employer_name"]
+    @employment ? @employment["employer_name"] : I18n.t("cbv.payment_details.show.unknown")
```

Comment:
Why not use those fancy new methods here?

```suggestion
    return I18n.t("cbv.payment_details.show.unknown") unless has_employment_data?

    @employment["employer_name"]
```

Reply:
Ah, I didn't consider that. I mean, yeah, it's kind of awkward. (You can always feel free to push back on my suggestions if you think they'll suck, heh.) The most important thing to me is that there is some test coverage of how these pages render with different subsets of data so we don't accidentally break it in the future if we were to improve the way these methods work later.